src/java/org/apache/cassandra/cql3/statements/schema/CreateTableStatement.java

```diff
@@ -479,12 +479,21 @@ public String defaultCompactValueName()
         }
     }
 
+    public static TableMetadata.Builder parse(String cql, String keyspace, String table, Types types, UserFunctions userFunctions)
```

Comment:
I feel that I am missing something. Why did you change that code?

Reply:
@blerer the original version of that method is still there. We just create this one in order to be able to specify custom keyspace and table name with types and functions. If we parse CQL for old table, we just want to be able to put there keyspace and table name etc for new table.